templates/default/sql/dml/undo/0.10.0__undo.sql.erb

```diff
@@ -24,3 +24,5 @@ DELETE FROM `hopsworks`.`variables` WHERE `id`="hive_scratchdir_delay";
 DELETE FROM `hopsworks`.`variables` WHERE `id`="hive_scratchdir_cleaner_interval";
 DELETE FROM `hopsworks`.`variables` WHERE `id`='admin_email';
 
+DELETE FROM `hopsworks`.`variables` WHERE `id`="hdfs_ui_ip";
```

Comment:
These should go in the `0.10.0.sql.erb`
In this file you should add the `REPLACE ....` to add the properties back in case someone wants to rollback to a previous version.

Reply:
Fixed